lib/screens/lista_contatos.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:bytebank/dao/ContatoDao.dart';
+import 'package:bytebank/models/contato.dart';
+import 'package:bytebank/screens/formulario_transferencia.dart';
+import 'package:flutter/material.dart';
+
+import 'formulario_contato.dart';
+
+const _tituloAppBar = 'Transferir';
+
+class ListaContatos extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return _ListaContatosState();
+  }
+}
+
+class _ListaContatosState extends State<ListaContatos> {
+  final List<Contato> _contatos = List();
+  final ContatoDao _dao = ContatoDao();
+
+  //TODO todas as buscas iniciais para fazer load na tela estou considerando essa função, existe alguma outra maneira recomendada?
+  @override
+  void initState() {
+    super.initState();
+    _buscaContatos();
+  }
+
+  Future _buscaContatos() async {
+    final contatosEncontrados = await _dao.todos();
+    setState(() {
+      this._contatos.addAll(contatosEncontrados);
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(_tituloAppBar),
+      ),
+      body: ListView.builder(
+        itemCount: _contatos.length,
+        itemBuilder: (context, posicao) {
+          final contato = _contatos[posicao];
+          return Container(
+            height: 100,
+            child: Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: Card(
```

Comment:
Why are you using a Card?
For the rounded edges?
Container allows decorations as well.

Reply:
I'm using because is a simple way to creating a item list element, I'm thinking to use only the Card Widget, what think?

```
return Card(
    child: ListTile(
        onTap: () => _vaiParaFormularioTransferencia(contato),
        title: Text(
        contato.nome,
        style: TextStyle(fontSize: 24.0),
        ),
        subtitle: Text('${contato.numeroConta}'),
    ),
);
```

Reply:
Ah I get it now 👍